extensions/yaml/package.json

```diff
@@ -13,6 +13,18 @@
   },
   "contributes": {
     "languages": [
+      {
+        "id": "compose",
+        "aliases": [
+          "Compose",
+          "compose"
+        ],
+        "filenamePatterns": [
+          "*compose*.yml",
```

Comment:
Discussed in the chat, but is this case-sensitive?

Reply:
I don't have a linux VM to test this on at the moment, but looking at the code it seems that we `toLower` the pattern so it should be case-insensitive.

https://github.com/microsoft/vscode/blob/e8760a5d6c0e09ab692e5b3e414eca018745490f/src/vs/base/common/mime.ts#L174-L182